README.md

```diff
@@ -30,51 +30,67 @@ https://github.com/conda-forge/miniforge/#download
 
 ## Compile LPython
 
-Create a Conda environment using the preexisting environment.yml file:
-
-    conda env create -f environment.yml
-    conda activate lp
-
 Install required packages (Linux - 64 bit):
 
-    sudo apt install binutils-dev zlib1g-dev
+```bash
+sudo apt install binutils-dev zlib1g-dev
```

Comment:
We should add `zlib` into the Conda environment, that should take care of it.

The `binutils` are currently needed for stacktraces on Linux --- I'd like to get rid of it and move fully to the same approach as macOS is using which does not require binutils, but I didn't have time to finish it yet.

Reply:
> We should add zlib into the Conda environment, that should take care of it.

Good point @certik, fixed it.

> The binutils are currently needed for stacktraces on Linux --- I'd like to get rid of it and move fully to the same approach as macOS is using which does not require binutils, but I didn't have time to finish it yet.

That makes sense, I can work on this and create a separate PR if it is okay with you. :)